example/keras/Dockerfile.example.keras.cntk_backend

```diff
@@ -0,0 +1,14 @@
+FROM aiplatform/pai.run.cntk
+
+# install git
+RUN apt-get -y update && apt-get -y install git
+
+# install Keras using pip
+RUN pip install keras cntk-gpu
+
+# clone Keras examples
+RUN git clone git clone https://github.com/keras-team/keras.git 
+
+ENV KERAS_BACKEND cntk
+
+WORKDIR /keras/examples
```

Comment:
The `WORKDIR` in base image is `/root`, so keras is cloned into `/root/keras`?

Reply:
updated, but workdir in latest pulled aiplatform/pai.run.tensorflow image is `/`